packages/java/endpoint/src/main/java/dev/hilla/EndpointController.java

```diff
@@ -158,6 +158,9 @@ public void registerEndpoints() {
             endpointBeans.forEach((name, endpointBean) -> endpointRegistry
                     .registerEndpoint(endpointBean));
         }
+
+        // Usage statistics
+        HillaStats.report();
```

Comment:
The only question I have: 
Does it make sense to wrap this method call in a `try { } catch(Throwable t) { log(..., t); }` to make sure a runtime error in `UsageStatistics.markAsUsed` is not affecting the application runtime behavior? Or do we intentionally want it to fail in case it wasn't able to gather the statistics?

Reply:
The `UsageStatistics.markAsUsed()` is called from multitude of places (also from Flow and from Vaadin Components), and it's always called directly (without being wrapped in a `try{} catch` block). I'd therefore suggest to follow that pattern; if need be, the `UsageStatistics.markAsUsed()` function itself should handle exceptions internally.